README.md

```diff
@@ -1,5 +1,14 @@
 # GitHub Action for running pylint commands
 
+Example workflow:
+
+```
+...
+- uses: cclauss/GitHub-Action-for-pylint@0.7.0
+- run: pip install pylint; pip install -r requirements.txt ; pylint **/*.py
```

Comment:
This would fail if there is no requirements.txt file.

Reply:
Indeed it would. The README right now is missing how to actually use this action though. See issue #3, users have no idea what to do.